bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, module, is_ipython):
         self.pythonization = pythonization
 
         self._is_ipython = is_ipython
+        self._original_preload = original_preload
```

Comment:
Why is it necessary to store these in the facade?

Reply:
I need this data to be available in one test, see https://github.com/root-project/roottest/pull/910/commits/185b300951af189ab3c43e403a7bf17f7a08b931 . If there is a better way of passing them through, please let me know.

Reply:
I was thinking we could move all this out of PyROOT and just do it when we need it for our tests, otherwise this becomes part of what's done whenever someone imports ROOT.

For roottest Python tests, there's a `common.py` module ([here](https://github.com/root-project/roottest/blob/master/python/common.py)) where we could potentially define these `original_preload` and `cleaned_preload`, and then access them from the tests, so everything stays in roottest.

Reply:
We have to execute before and after `import ROOT` though, to get both versions. But maybe that's possible, it's really only that one test...

Reply:
We can place what is now in `ROOT/__init__.py` in `common.py`. `common` is already imported from `PyROOT_regressiontests.py`, we just have to make sure we import it before ROOT. And then in the test we access the variables that we imported from common instead of from ROOT.

Reply:
No, I think we have to keep the modification of `LD_PRELOAD` in `__init__.py` because we need it for all `import ROOT`, not only the unit tests.

Reply:
Why do we need everyone who imports ROOT to check if asan libraries need to be taken out of LD_PRELOAD? Isn't this something we just need for our builds and tests?

Reply:
In principle, maybe. But how do activate code only for our tests? Also users or developers may want to test instrumented builds with their scripts, to make sure they are correct.

Reply:
Moved the variables to the one test that needs them; tweaking `LD_PRELOAD` is still done on `import ROOT`